tests/test_helper.js

```diff
@@ -0,0 +1,25 @@
+import { JSDOM } from 'jsdom';
+
+const jsdom = new JSDOM('<!doctype html><html><body></body></html>');
+const { window } = jsdom;
+
+function copyProps(src, target) {
```

Comment:
It would be nice to add a small commentary what each section is suppose to do and why :)